node-via-nvm/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+NODE_VERSIONS=("${NODE_VERSIONS}")
+BOLD='\033[0;1m'
+CODE='\033[36;40;1m'
+RESET='\033[0m'
+
+printf "$${BOLD}Installing nvm!$${RESET}\n"
+
+export NVM_DIR="${INSTALL_PREFIX}/nvm"
+
+output=$(curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/${NVM_VERSION}/install.sh | bash)
```

Comment:
```suggestion
# top
NVM_VERSION='${NVM_VERSION}'

# here
output=$(curl -o- "https://raw.githubusercontent.com/nvm-sh/nvm/$${NVM_VERSION}/install.sh" | bash)
```

Reply:
This will fail silently on network errors, see:

```sh
bash-5.2$ false | bash; echo $?
0
```

Here `bash` masks the exit code of curl.

We also don't capture stderr output here, so the `Failed to install` echo below will potentially be empty or not contain the actual failure. Maybe OK but could be confusing.

Reply:
How can this be fixed in the code?

Reply:
One option would be:

```sh
script="$(curl -o- "[url]")"
# handle error...
output="$(bash <<<"$script")"
# handle error...
```

Reply:
Updated curl with `-sS` so that we stay silent, but show error if there is one. That way the `2>&1` won't dirty the script output, but it'll still let us show an error if we failed and print `$script`.

Reply:
Thanks!

Reply:
Saw your change, and you also need to handle the error after curl, i.e. `if [ $? ..` between script and output.